### DISCUSSION TITLE
终于入驻绳网啦

### DISCUSSION THREAD

**Discussion by HeroBrine-c** [share121/inter-knot#1350] [其他]
### 在线接委托 

本匠宗旨：### 为人民服务，励志成为传奇绳匠_
这是本匠的入驻宣言：[ı||ııı||ı|ıı||ı| 127”](https://www.bilibili.com/video/BV1hq4y1s7VH/?spm_id_from=333.337.search-card.all.click&vd_source=292a736272998f2c0ba897d68fcc9f90)
 联系方式：@HeroBrine-c
有特别困难的委托，交给我好了，包您满意的doge

本匠uid:30******

_备注：全天24小时不在线_

**Comment by lllliufeng** [comment]
> [!IMPORTANT]
> [戳我](https://www.bilibili.com/video/BV1GJ411x7h7/)防骗

**Comment by su2569** [comment]
.
